package.json

```diff
@@ -44,6 +44,7 @@
     "@types/yargs": "17.0.12",
     "@typescript-eslint/eslint-plugin": "5.33.0",
     "@typescript-eslint/parser": "5.43.0",
+    "docsify-cli": "^4.4.4",
```

Comment:
the build is failing because of the `^`, we like to pin our version dependencies